about/roles.md

```diff
@@ -84,12 +84,16 @@ Should a coach not be available to help with an issue, students can also reach
 out to remote coaches (see below). Coaches can also be mentors, which is
 explained below.
 
-<h3 id="mentor">Mentor</h3>
+<h3 id="mentor">Mentor / Project Maintainer</h3>
 
 Mentors are experts on the Open Source project the team is working on. Ideally
 they are decision makers, or work closely with decision makers, so they can set
 general goals, give directions and provide feedback.
 
+At the beginning of summer Mentors will help students design a roadmap for the 
+project and throughout the project be available to review pull requests, help
+with concepts and generally support the team.
+
 Since students will be supported by coaches in their day-to-day work, the
```

Comment:
This wasn't in the original issue, but could you replace “will be” with “are”? I think most of the page is written in the present tense, so it's good to keep it consistent. Thanks!